### PR TITLE
Assign bridge IPs for the hosts

### PR DESCRIPTION
**Pull request by hectorj2f**
If you choose a brigde network model, we are able to declare a range of ips to be used and choose a free one for our hosts.

ping @teemow

### CODE REVIEW

**Comment by hectorj2f** [comment]
ping @teemow

File: config.yaml.dist [hunk 1]
```diff
@@ -8,6 +8,9 @@ network:
   ip_range:
     start: 10.0.3.31
     end: 10.0.3.70
+  bridge_ip_range:
+    start: 12.0.3.31
```
Comment (teemow) [inline config.yaml.dist]: `12.0.3.0` is not a private ip range. let's use something like `192.168.10.1/24`. and there is no need to use `start`, `end` here. so let's use the full CIDR notation that is used in the networkd template.

File: pxemgr/ipxe_handlers.go [hunk 2]
```diff
@@ -105,6 +106,14 @@ func (mgr *pxeManagerT) maybeCreateHost(serial string) *hostmgr.Host {
 			}
 		}
 
+		if mgr.config.Network.NetworkModel == "bridge" || mgr.config.Network.NetworkModel == "bond_bridge" {
+			host.BridgeIP = mgr.getNextBridgeIP()
```
Comment (teemow) [inline pxemgr/ipxe_handlers.go]: Let's not use different ips for the bridges. that won't work ootb anyway. I'd remove this part. You should use flannel or something similar for this.

**Comment by hectorj2f** [comment]
Our deployment solution has changed since then, we can close this PR.

**Pull request closed without merging**